docs/features/native_functions.md

```diff
@@ -0,0 +1,31 @@
+# Native Functions
+
+```js
+intInBase10(10) // Takes in an int and returns the string version.
+round(10.5) // Rounds a float to the nearest value.
+floor(10.5) // Floors a float to the nearest value.
+ceil(10.4) // Ceilings a float to the nearest value.
+charCode(\{a}) // Returns the code for a character.
+intCode(32) // Returns the chacter associated for an int.
+charAt(1, "abc") // Returns the character at an index of a string, takes in an int and a string and returns a maybe.
```

Comment:
Could clarify that it returns a `maybe[char]`

Reply:
Added.